README.md

```diff
@@ -200,4 +201,14 @@ NULL ...
 ### Internals
 ![](./assets/internals.png)
 
+## Release versioning
+The release version of django-spanner in [setup.py](./setup.py) SHOULD follow this form
```

Comment:
I would still like you to add a reference to some documentation, either a PEP or something more Django-specific that expects us to follow this convention.

I looked through [PEP 440](https://www.python.org/dev/peps/pep-0440) and I didn't find anything relevant but you may have a different reference.

Reply:
Could we perhaps dig up the reference after merging this PR and releasing? I got the information and guidance from Tim whom I'll consult to dig up the reference.

Reply:
The release is still pending approval so if we can complete the README in the meantime, that would be great.